content/tutorials/integration-tutorials/keras.md

```diff
@@ -7,23 +7,25 @@ title: Keras
 ---
 
 {{< cta-button colabLink="https://colab.research.google.com/github/wandb/examples/blob/master/colabs/keras/Use_WandbMetricLogger_in_your_Keras_workflow.ipynb" >}}
-Use Weights & Biases for machine learning experiment tracking, dataset versioning, and project collaboration.
+Use Weights & Biases for machine learning experiment tracking, dataset
+versioning, and project collaboration.
 
 {{< img src="/images/tutorials/huggingface-why.png" alt="" >}}
 
-This Colab notebook introduces the `WandbMetricsLogger` callback. Use this callback for [Experiment Tracking]({{< relref "/guides/models/track" >}}). It will log your training and validation metrics along with system metrics to Weights and Biases.
-
+This notebook introduces the `WandbMetricsLogger` callback. Use this callback
+for [Experiment Tracking]({{< relref "/guides/models/track" >}}). It will log
+your training and validation metrics along with system metrics to Weights and
+Biases.
 
 ## Setup and Installation
 
-First, let us install the latest version of Weights and Biases. We will then authenticate this colab instance to use W&B.
-
+First, let us install the latest version of Weights and Biases. We will then
```

Comment:
```suggestion
In the notebook, install Weights and Biases and sign in.
```
It's better to avoid "let us" and "we" in favor of the simple imperative. I believe both of these code blocks should be tagged up as notebooks.